### PR TITLE
Display nicer balances.

### LINKED ISSUE #7: Formatting of large balances
Super large balances overflow the box and look awkward. Might be worth introducing some simple short formatting if there is more than 1M of tokens (for instance `2.3M`). The full amount should be part of `title` attribute.

### PR DESCRIPTION
**Pull request by tomusdrw**
Resolves #7

### CODE REVIEW

File: src/screens/EthToSub.tsx [hunk 1]
```diff
@@ -219,6 +251,8 @@ export default styled(EthToSub)`
 	.balance{
 		font-weight: 800;
 		font-size: large;
+    text-overflow: ellipsis;
+    overflow: hidden;
```
Comment (Tbaut) [inline src/screens/EthToSub.tsx]: using tabs
```suggestion
	text-overflow: ellipsis;
	overflow: hidden;
```

File: src/screens/EthToSub.tsx [hunk 2]
```diff
@@ -200,6 +214,24 @@ const EthToSub = ({ className, ethProvider } : Props) => {
 	);
 };
 
+function toEthBalance(v: BigInt) {
```
Comment (Tbaut) [inline src/screens/EthToSub.tsx]: Could we move these 2 `toEthBalance` and `toBalance` to `./util` as well? I feel like if there ever is a new page, we'll need them there as well :)

**Review by Tbaut — APPROVED** [review]
Just a couple nits.

File: src/screens/EthToSub.tsx (hunk 2 above)
Comment (tomusdrw) [reply src/screens/EthToSub.tsx]: Updated.

**Review by Tbaut — APPROVED** [review]
Nice thanks, untested ack, I'll play with it now anyway.

**Pull request merged**